standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/MetaStoreTServerSocket.java

```diff
@@ -18,19 +18,37 @@
 
 package org.apache.hadoop.hive.metastore;
 
-import java.net.SocketException;
+import java.util.Objects;
 
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hive.metastore.conf.MetastoreConf;
 import org.apache.thrift.transport.TServerSocket;
 import org.apache.thrift.transport.TSocket;
 import org.apache.thrift.transport.TTransportException;
 
 /**
- * TServerSocketKeepAlive - like TServerSocket, but will enable keepalive for
- * accepted sockets.
- *
+ * Give HMS a chance to customize the accepted sockets, such as
+ * set the thrift max message size.
  */
-public class TServerSocketKeepAlive extends TServerSocket {
-  public TServerSocketKeepAlive(TServerSocket serverSocket) throws TTransportException {
+public class MetaStoreTServerSocket extends TServerSocket {
```

Comment:
since it's super simple, why don't we use an anonymous class similar to ThriftBinaryCLIService?

Reply:
done